### ISSUE TITLE
Issues with Compiling

### ISSUE THREAD

**Issue by DeNarr** [CleverRaven/Cataclysm-DDA#12885]
The current experimental build is failing when 'make TILES=1 RELEASE=1' is ran.

In file included from src/speech.cpp:3:0:
src/rng.h: In instantiation of ‘const V& random_entry(const C&) [with C = std::vector<SpeechBubble>; V = SpeechBubble]’:
src/speech.cpp:44:46:   required from here
src/rng.h:45:38: error: missing initializer for member ‘SpeechBubble::text’ [-Werror=missing-field-initializers]
         static const V default_value{};
                                      ^
src/rng.h:45:38: error: missing initializer for member ‘SpeechBubble::volume’ [-Werror=missing-field-initializers]
cc1plus: all warnings being treated as errors
make: *** [obj/tiles/speech.o] Error 1

**Comment by Coolthulhu** [comment]
Confirming for gcc 4.9.1, it doesn't like the default initialization either.

**Comment by karlnp** [comment]
Hmm. It works through my CMake process, which uses g++. I'll reset it to use gcc and report back.

**Comment by tvm1** [comment]
It throws warning on 4.8.2 and builds but fails all the jenkins builds (except linux-32).

**Comment by drbig** [comment]
Reporting successful build & very minimal test run with:
```
time make -j4 NATIVE=linux64 LOCALIZE=0 LUA=1 TILES=1 RELEASE=1
g++ (GCC) 5.1.0
```

**Comment by drbig** [comment]
Also
```
clang version 3.6.1 (tags/RELEASE_361/final)
```
Builds and run very fine too.

**Comment by karlnp** [comment]
gcc chokes hard for me. I'm poking at it in between actual $DAYJOB work.

**Comment by kevingranade** [comment]
I went ahead and merged since I didn't want the build to stay broken, happily it seems to have fixed the issue for both gcc 4.7.3 and mingw-w64-whatever, please post if it isn't working on some compiler.

**Comment by karlnp** [comment]
Oh, cool, I was just staring at the build progress. I'll pull and report back.
